### ISSUE TITLE
Number coerced to Int

### ISSUE THREAD

**Issue by oligus** [vuex-orm/vuex-orm-graphql#32]
With a schema using `ID` type, fetching a record via `fetch` coerces number type to `Int`.

Example query:
```
query PriceLists($maoId: ID) {
  priceLists(filter: {maoId: $maoId}) {
    nodes {
      id
      fromDate
      toDate
      active
    }
  }
}

variable:
{
  "maoId": 148
}
```

Fetching with:
`const result = await PriceListsModel.fetch({maoId: 148})`

Results in:
```
operationName: "PriceLists"
query : "query PriceLists($maoId: Int!) {
  priceLists(filter: {maoId: $maoId}) {
    nodes {
      id
      fromDate
      toDate
      active
      __typename
    }
    __typename
  }
}"

variables:{maoId: 148}
``` 

The coercion is probably made here: https://github.com/vuex-orm/vuex-orm-graphql/blob/544b2c91b8e5fbab74f329f7a1b8b0e1d005f1b7/src/graphql/query-builder.ts#L221-L222

**Comment by phortx** [comment]
Thanks for reporting, I will take a look :)

**Comment by phortx** [comment]
https://github.com/vuex-orm/vuex-orm-graphql/pull/33

**Comment by phortx** [comment]
@oligus I pushed a fix in this PR, could you test that via `yarn link`

**Comment by oligus** [comment]
@phortx Unfortunately, this did not work, it fails to get schemaField for the argument I guess?

```
if (signature) {
   const schemaField = Context.getInstance().schema!.getType(model.singularName).fields!
   .find(f => f.name === key);

   console.log('schemaField:', schemaField)
   console.log('args:', args)
   console.log('value:', value)
```

Yields:
``` 
schemaField: undefined
args: {maoId: 148}
value: 148
```

**Comment by phortx** [comment]
Could it be, that the type doesn't have a field of the name `maoId` and this is only used as argument?

In this case it gets somewhat tricky ...

**Comment by oligus** [comment]
Both yes and no. I have two separate fields, the `priceLists` field which I'm querying and the singular `priceList` field which is a field and a type.

The `nodes` in price lists are of type `PriceList` that doesn't seem to show up in the introspection.

PriceLists:
``` 
{  
	"name":"PriceLists",
	"description":"Price list",
	"fields":[{  
		"name":"count",
		"description":"Record count",
		"type":{  
			"name":"Int",
			"kind":"SCALAR"
		}
	},
	{  
		"name":"nodes",
		"description":"Records",
		"type":{  
			"name":null,
			"kind":"LIST"
		}
	}],
	"inputFields":null
}
```

PriceList:
```
{  
	"name":"PriceList",
	"description":"Price list",
	"fields":[ {  
		"name":"id",
		"description":"Price list ID",
		"type":{  
			"name":"ID",
			"kind":"SCALAR"
		}
	},
	{  
		"name":"fromDate",
		"description":"From date",
		"type":{  
			"name":"String",
			"kind":"SCALAR"
		}
	},
	{  
		"name":"toDate",
		"description":"From date",
		"type":{  
			"name":"String",
			"kind":"SCALAR"
		}
	},
	{  
		"name":"musicUsageArea",
		"description":null,
		"type":{  
			"name":"MusicUsageArea",
			"kind":"OBJECT"
		}
	},
	{  
		"name":"priceRows",
		"description":null,
		"type":{  
			"name":"PriceRows",
			"kind":"OBJECT"
		}
	},
	{  
		"name":"active",
		"description":null,
		"type":{  
			"name":"Boolean",
			"kind":"SCALAR"
		}
	}]
	"inputFields":null
}
```

**Comment by phortx** [comment]
I pushed a fix to the PR for the case that an argument is not in the fields. But I don't think that will help you alot in this case because there seems to be an "error" in the schema. The Type of `PriceLists` should be a `PriceListConnection`. This may be confusing the plugin, because it expects those query fields to be of a Connection type.

When you take a look in the file `mock-data.js` you see an example introspection result which is the expected layout of a schema with Connection and Filter Types. This is generated by a real GraphQL Server.

(OffTopic: This mock response will be replaced by a real graphql server in the future)

**Comment by oligus** [comment]
Thanks! 

It worked out, as you can see the query correctly identifies the type:

```
query PriceLists($maoId: ID!) {
  priceLists(filter: {maoId: $maoId}) {
    nodes {
      id
      fromDate
      toDate
      active
      __typename
    }
    __typename
  }
}

variables: {maoId: 148}
```

**Comment by phortx** [comment]
Wow! Nice :) 

I'm surprised that this plugin works at all without having Connections in the Schema :D This will cause the auto detection of the connection query mode to fail, which is ok, but I thought there would be many other issues. I'm happy to hear that this is not the case :)

I will release a new version in a few minutes :)

**Comment by phortx** [comment]
Published [1.0.0-rc.9](https://github.com/vuex-orm/vuex-orm-graphql/releases/tag/v1.0.0-rc.9)
